tests/std/tests/P0896R4_ranges_alg_for_each/test.cpp

```diff
@@ -72,4 +72,4 @@ struct instantiator {
     }
 };
 
-template void test_in<instantiator>();
+template void test_in<instantiator, const int>();

```

Comment:
Pre-existing: Since `for_each` (and `for_each_n`) allow the callable object to modify elements, should there be instantiation test coverage for that? No change requested.

Reply:
I'm not greatly concerned. It wouldn't hurt, but the algorithm itself never so much as sees an element so there isn't much to go wrong. In any case, such additional testing shouldn't be part of this PR. Feel free to file an issue if you are motivated to do so and I'll add it to my queue ;)

Reply:
That's a good point, it would be pretty hard for the algorithm to add constness to the elements.